daemon/openrazer_daemon/hardware/accessory.py

```diff
@@ -54,6 +54,18 @@ class RazerBaseStationChroma(_RazerDeviceBrightnessSuspend):
     DEVICE_IMAGE = "https://rzrwarranty.s3.amazonaws.com/145dcc47f9f9d33b0bd07b066364704160f45e87b756d690b203decec7d1e87c.png"
 
 
+class RazerMouseDock(RazerBaseStationChroma):
```

Comment:
This is a distinct device from RazerBaseStationChroma so please base on `_RazerDeviceBrightnessSuspend` directly and copy the USB_VID over (and add a docstring like in the other classes)